vm-zone-move/README.md

```diff
@@ -0,0 +1,51 @@
+# Failover scripts
+
+During zone down scenario, the VM cannot be accessed. These scripts will bring back the vm in a different zone or region depending upon the type of disks (ZRS/LRS) attached to the vm. Affectively moving the VM. 
+
+## How to Use? 
+
+* Ensure that python is installed on the machine where these scripts are going to be run. 
```

Comment:
Can you provide commands instead of bullet points?

Reply:
can add the command but having example is fine i believe